test/parallel/test-os.js

```diff
@@ -101,7 +101,7 @@ console.error(interfaces);
 switch (platform) {
   case 'linux':
     {
-      const filter = function(e) { return e.address == '127.0.0.1'; };
+      const filter = function(e) { return e.address === '127.0.0.1'; };
       const actual = interfaces.lo.filter(filter);
```

Comment:
The arrow function will work in the `linux` part, but I don't believe it can (at least not in any obvious way) in the second instance (in the `win32` portion) where the line it needs to be put into is longer. I'm reluctant to make them different from each other. Seems clearer to me to have them both be the same, even if it means an extra line in one case. Since the comment is labeled as a `nit` and ends with a question mark, I'm going to take that to mean that it's OK for this to land without that suggestion.